images/ceph/Makefile

```diff
@@ -79,39 +80,38 @@ do.build: generate-csv-ceph-templates
 		$(TEMP);\
 	fi
 	@rm -fr $(TEMP)
-	@$(MAKE) -C ../.. crds # revert changes made to the crds.yaml file during the csv-gen sequence
 
-generate-csv-ceph-templates: $(OPERATOR_SDK) $(YQ)
-	@if [ ! "$(INCLUDE_CSV_TEMPLATES)" = "" ]; then\
-		if [ "$(GOARCH)" = amd64 ]; then\
```

Comment:
Parallel builds will no longer step on each other since they can stage the CSV generation results, so this check is no longer necessary.